lib/prisma.ts

```diff
@@ -168,7 +172,8 @@ export default {
   getDateQuery,
   getTimestampIntervalQuery,
   getFilterQuery,
-  getFunnelQuery,
+  toUuid,
   parseFilters,
+  getFunnelQuery,
```

Comment:
This should probably be moved inside `getFunnel.ts` since it probably won't be used anywhere else.

Reply:
move relational / CH function into getFunnel.ts